app/src/lib/api/mailchimp.ts

```diff
@@ -47,3 +51,95 @@ export async function updateMailchimpTags(
     return { success: false, error }
   }
 }
+
+/**
+ *
+ * @param {FormData} data - Must contain:
+ *   - `email` (string): The user's email address.
+ */
+export async function addContactToList(data: FormData) {
```

Comment:
what I was thinking is this

```suggestion
export async function addContactToList(email: string) {
```